graphite_api/functions.py

```diff
@@ -2711,14 +2711,18 @@ def smartSummarize(requestContext, seriesList, intervalString, func='sum'):
 
     # Adjust the start time to fit an entire day for intervals >= 1 day
     requestContext = requestContext.copy()
+    # ISSUE #51: Added tzinfo to requestContext in graphite_api/app.py:328
+    tzinfo = requestContext.get('tzinfo', None)
```

Comment:
You can directly do `tzinfo = requestContext['tzinfo']`: it's guaranteed to be in the request context (and the comment above is unnecessary)